examples/web/menu.html

```diff
@@ -0,0 +1,87 @@
+<!-- The software contained in this file is free and unencumbered software
```

Comment:
Can you add a first sentence saying:  `Created by RidgeRun, 2020` to all headers?